components/Footer/index.js

```diff
@@ -0,0 +1,67 @@
+import Link from "next/link";
+import React from "react";
+import styled from "styled-components";
+
+export default function Footer() {
+  return (
+    <StyledFooter>
+      <div class="wrapper">
+        <div className="top_nav"></div>
+      </div>
+      <div class="bottom_nav">
+        <LinkList>
+          <NavLink>
+            <Link href="/">Home</Link>
+          </NavLink>
+          <NavLink>
+            <Link href="/interview">Events</Link>
+          </NavLink>
+          <NavLink>
+            <Link href="/about">Discover</Link>
+          </NavLink>
+        </LinkList>
```

Comment:
Links shouldn't lead to a `404` page. Build the navbar when you've already built a page that you can actually link to. As long as you don't have a page to link to, don't include dead links. Currently, `/interview` and `about` lead to non-existing pages.

Reply:
Thank you. It helped me a little to give a rough styling to the footer but forgot to remove them. All gone now.